services/provider/server/storageclaim.go

```diff
@@ -0,0 +1,95 @@
+package server
+
+import (
+	"context"
+	"crypto/md5"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+
+	ocsv1alpha1 "github.com/red-hat-storage/ocs-operator/api/v1alpha1"
+
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type storageClassClaimManager struct {
+	client    client.Client
+	namespace string
+}
+
+func newStorageClassClaimManager(ctx context.Context, cl client.Client, namespace string) (*storageClassClaimManager, error) {
+	return &storageClassClaimManager{
+		client:    cl,
+		namespace: namespace,
+	}, nil
+}
+
+func getStorageClassClaimName(consumerUUID, storageClassClaimName string) string {
+	var s struct {
+		StorageConsumerUUID   string `json:"storageConsumerUUID"`
+		StorageClassClaimName string `json:"storageClassClaimName"`
+	}
+	s.StorageConsumerUUID = consumerUUID
+	s.StorageClassClaimName = storageClassClaimName
+
+	claimName, err := json.Marshal(s)
+	if err != nil {
+		klog.Errorf("failed to marshal a name for a storage class claim based on %v. %v", s, err)
```

Comment:
```suggestion
		klog.Errorf("failed to marshal a name for a storage class claim based on %v. %v", s, err)
```

Reply:
what change is expected here? am not able to make out what change is required here.